ArcGISRuntimeSDKQt_CppSamples/Scenes/SyncMapViewSceneView/SyncMapViewSceneView.cpp

```diff
@@ -25,9 +25,10 @@
 #include "Map.h"
 #include "MapQuickView.h"
 #include "MapTypes.h"
-#include "TaskWatcher.h"
 #include "Viewpoint.h"
 
+#include "QFuture"
```

Comment:
Should this be `#include <QFuture>`?
That's how I'm used to seeing included Qt components like here: 
https://github.com/Esri/arcgis-maps-sdk-samples-qt/blob/v.next/ArcGISRuntimeSDKQt_CppSamples/Layers/AddCustomDynamicEntityDataSource/AddCustomDynamicEntityDataSource.cpp#L48

Reply:
yeah good catch, it changes the the preprocessor behavior.